draft-ietf-masque-connect-udp-listen.md

```diff
@@ -246,6 +262,7 @@ back to the client.
             <--------  STREAM(44): HEADERS
                          :status = 200
                          capsule-protocol = ?1
+                         proxy-public-address = 192.10.11.45:3030
```

Comment:
IPv4 examples need to use the prefixes reserved for documentation by RFC 5737. I'd also add IPv6. The documentation prefix for v6 is 2001:db8::/32. I'd also use an ephemeral port. All together:

```suggestion
                         proxy-public-address = 192.0.2.45:54321,[2001:db8::1234]:54321
```